doc/Contributing.md

```diff
@@ -0,0 +1,52 @@
+
+# Code development #
+
+Anything that does not work or exist needs fixing. You need to know C++, and make commits using the [code style](CodeStyle.md) of the project, and provide Doxygen comments with it.
+
+The code should build out of the box at a Linux system. For Windows, it is currently a little more difficult. See [BuildProgramVisualStudio](BuildProgramVisualStudio.md) for more details.
+
+If you need graphics that do not exist yet, just make some stub graphics. It does not matter how ugly they are.
+
+Submissions take the form of pull requests via Github.
+
+
+# Graphics development #
+
+We need LOTS of graphics. Most of the graphics are created using the [Blender program](http://www.blender.org). There are textures and templates in our repository that you should use, so everything gets a uniform look and a standardized format so it can be automagically processed without much effort.
+
+Note that the GPL v2 license of the project requires you to make all your source files publicly available, so others can build on your work. Some source files may not allow such publication, please check the license conditions beforehand.
+
+## RCD files ##
+For future contributions with graphics and item properties, an alternative route exists (in theory so far, nobody has tried it yet). On startup, the program reads RCD files to load its graphics and item properties. The point of this is that everybody can create such a file, and thus make his own extensions.
+
+Currently the **rcdgen** converter program exists (as in, you can download its source, and build it). There is also documentation what you can express in it, and the source files of the used graphics from the project data are available.
+
+# Where should you work on #
+
+In short, _anything that does not exist yet_. Since that is a bit vague, below is a little list
+
+  * **Security** (needs benches, and/or lights, security guards walking around, guests that spot guards)
+  * **Handymen** (needs growing grass, rides that break down, a dispatch system calling handy men for repair and maintenance, handy men walking around, and mowing the grass)
+  * **Litter** (needs litter at paths, litter bins, handy men collecting litter)
+  * **Park border** (needs park gate, option to buy new land)
+  * **Guests finding their way through the park** (needs guests understanding what's available and what they want, some sort of advantage for owning a park map)
+  * **User finding his way** (UI to display the park in small scale, windows listing rides, statistics, problem areas)
+  * **Popularity** (needs a measure of how good the park is, feedback coupling to number of guests arriving (but up to some point so it doesn't spiral out of control), advertising)
+  * **Load and save of games** (discussion: https://groups.google.com/forum/#!topic/freerctgame/h1iIJ-DLJWI )
+  * **Scenarios**
```

Comment:
Add Finances / Money transactions ?  Finances needs to be properly embedded in the code, currently pretty much everything is free, and can be done no matter how bankrupt you are :)